test/runtests.jl

```diff
@@ -13,7 +13,6 @@ const u = UnitfulAstro
         @test 365.25*u.d == 1*u.yr
         @test 1*u.ly == 1*u.c*u.yr
         @test 1*u.pc == 1*u.AU/u.arcsecond
-        @test 1*u.angstrom == 1e-10*u.m
```

Comment:
The unit is no more defined by this package, nevertheless I think we may want to be sure it's available, as it's important for astronomical applications.